server/src/test/java/org/elasticsearch/search/profile/query/RandomQueryGenerator.java

```diff
@@ -169,6 +169,10 @@ private static QueryBuilder randomConstantScoreQuery(List<String> stringFields,
         return QueryBuilders.constantScoreQuery(randomQueryBuilder(stringFields, numericFields, numDocs, depth - 1));
     }
 
+    /**
+     * @deprecated See {@link CommonTermsQueryBuilder}
+     */
+    @Deprecated
```

Comment:
This is not needed (the deprecated annotation) and since it will generate a deprecation warning I wonder if this function should be removed entirely. Random queries that generates warning are difficult to handle in tests so I am leaning towards removing.